README.md

```diff
@@ -1,142 +1,181 @@
+| ARSS_IDENTITY_TYPE_OTP_AUTH            | The authentication domain of the ARSS user                                        | string  |
+| ARSS_IDENTITY_USER                     | The username of the ARSS user                                                     | string  |
+| ARSS_IDENTITY_USER_PWD                 | The password of the ARSS user                                                     | string  |
 
-A local SPID test environment is needed to fully test the application features. [spid-testenv2](https://github.com/italia/spid-testenv2) has been selected to achieve this goal.
+#### Optional environment variables
+
+The table lists the optional environment variables: if any of them is not set,
+the application will use its default value instead.
+
+| Variable name                       | Description                                                             | type   | default |
+| ----------------------------------- | ----------------------------------------------------------------------- | ------ | ------- |
+| ADMINISTRATION_SEARCH_RESULTS_LIMIT | The maximum number of administrations returned when performing a search | number | 30      |
+
+## Production deployments
 
-### Build steps
+An Azure pipeline is configured by the IO adminstrators
+to run on each push on the master branch.
 
-The *Dockerfile* and the `docker-compose.yaml` files used to build the application, are in the root of this repository.
+Each time a change is merged, a corresponding Docker image is automatically created on
+[DockerHub](https://cloud.docker.com/u/teamdigitale/repository/docker/teamdigitale/io-onboarding-pa-api).
 
-To build the application, run from the project root:
+The image is then deployed on Kubernetes clusters using a dedicated helm-chart;
+see the [production deployment repository](https://github.com/teamdigitale/io-infrastructure-post-config).
+
+The deployed backend is reachable at
+[https://api.pa-onboarding.dev.io.italia.it/]https://api.pa-onboarding.dev.io.italia.it/).
+
+## Test the application locally
+
+To build the application and starts the services, run from the project root:
 
 ```shell
-docker-compose up -d
+cp env.example .env
+# edit .env
+docker-compose up -d postgresql
+docker-compose up
 ```
 
->NOTE: the *docker-compose.yaml* file sets some environment variables that could be used to adapt the application features to specific needs. The *docker-compose.yaml* file sources variables from a *.env* file that should be placed in the root folder of this repository. An example file *.env.example* is distributed with the code. For your convenience, you may start copying this file, renaming it to *.env* and editing its values. More info about variables can be found in the dedicated paragraph, below in the readme.
+> The _docker-compose.yaml_ file sources variables from a _.env_ file that must be
+> placed in the root folder of this repository. An example file _.env.example_ is
```

Comment:
The example file has been renamed to `env.example`, here the name should be changed too.